base/src/main/java/com/smartdevicelink/proxy/rpc/OnDriverDistraction.java

```diff
@@ -73,6 +73,8 @@
  */
 public class OnDriverDistraction  extends RPCNotification {
 	public static final String KEY_STATE = "state";
+    public static final String KEY_LOCKSCREEN_DISMISSIBLE = "lockScreenDismissalEnabled";

```

Comment:
can you fix the indentations to match?

Reply:
Done, no change, mis-alignment is from previous code.